web/profiles/joinup/joinup.services.yml

```diff
@@ -1,4 +1,9 @@
 services:
+  cache_context.user.is_root:
```

Comment:
We don't need this context. Core always provide `cache_context.user.is_super_user`.

Reply:
Wow I didn't know about this one :sweat_smile: I see the implementations are virtually identical but mine has more strict typing :grin: 

I'll replace my implementation with the one provided by core.